spec/test_helper.rb

```diff
@@ -0,0 +1,21 @@
+class TestHelper
+  def self.get_first_row_event_from_client(ecco_client)
```

Comment:
I think that will lead to confusion regarding what is returned from the method. Otherwise I like the proposed change.

Maybe we could refactor so that start and stop is handled by the tests. We might need to take alook at how we can implement a non-blocking behavior in Ecco then though.